docs/supported-devices/index.rst

```diff
@@ -16,6 +16,9 @@ quite common amongst a variety of devices.
 Tested
 ======
 
+* Alfa
```

Comment:
Wrong syntax. This will create following list entry:

* Alfa - AP121F

But we want:

* Alfa
  * AP121F

Please make sure to add an empty line after "Alfa" to achive this